specifications/M1/verifiable.presentation.protocol.md

```diff
@@ -36,7 +35,7 @@ issued credentials.
 
 The VPP is based on Json-Ld message types. The CS Json-Ld context is:
 
-`https://w3id.org/tractusx/2023/cs/v1`
+`https://w3id.org/tractusx-trust/v0.8`
```

Comment:
The link ist still 404-ing, just checking if it's known.

Reply:
Yeah, we need to do that later.